src/vs/editor/contrib/inlineCompletions/browser/inlineCompletionContextKeys.ts

```diff
@@ -44,6 +47,7 @@ export class InlineCompletionContextKeys extends Disposable {
 		this._register(autorun(reader => {
 			/** @description update context key: inlineCompletionSuggestsIndentation, inlineCompletionSuggestsIndentationLessThanTabSize */
 			const model = this.model.read(reader);
+			if(!this.editor.getOption(EditorOption.inlineSuggest).suppressIndentationAcceptance) return;
```

Comment:
This is not a good fix, especially since it needs a reload when the setting value changes.